kibana.md

```diff
@@ -669,13 +669,13 @@ Enabling proxy auth passthrough will relax the requirement to provide a password
 
 This feature will work in all ReadonlyREST editions.
 
-Alternatively to typing in credentials in the standard login form, it is possible to create an authenticated Kibana session by passing a JWT token as a query parameter in a URL.
+Alternatively, to typing in credentials in the standard login form, it is possible to create an authenticated Kibana session by passing a JWT token as a query parameter in a URL.
```

Comment:
I don't understand this comma after alternatively

Reply:
you are right. I was too quick :P